src/test/java/com/github/fppt/jedismock/TestRedisServer.java

```diff
@@ -34,7 +36,7 @@ public void testBindErrorPort() throws IOException {
         RedisServer server = RedisServer.newRedisServer(100000);
         try {
```

Comment:
use check for exception here and in all the similar places

Reply:
OK